feat(FieldsBreakdown): Show panels with errors

gtk-grafana opened the pull request:
Fixes: https://github.com/grafana/logs-drilldown/issues/1343

https://github.com/user-attachments/assets/06cfe42e-299b-47dc-8996-6791a5e7d0a2

For reviewers:
Sharded results should show partial results in the panel.
When panels with errors are displayed the user can still get to the value breakdown, which will always show an error, however it is still useful for users that have partial sharded results.

matyax commented:
This is a great improvement overall, and takes the app one step closer to addressing this, but I don't think it fixes #1343, or it does it in a way that is still not fully clear to the user. Maybe we can get to a middle ground by:
- Changing the control to be slightly more verbose. Even with the tooltip, I think a "Show errors" control feels a bit try. An example:
  ![imagen](https://github.com/user-attachments/assets/a8af7a2e-2085-40ba-a0d9-1424820a43d1)
- Only show either version of "Show errors" when there are actually panels with errors, not all the time. When there are no errors it will be a noop and will use UI space without providing any service.
- Disable or gray out the "select" button on failed panels?
- We could catch the "max series" error in the querier and replace it with a better message. I can take this part.
- The field breakdown looks currently broken. One thing that we're missing there is a button to return to the fields/label breakdown.
  ![Captura de pantalla 2025-06-17 a la(s) 10 52 52](https://github.com/user-attachments/assets/c7fc8c21-0257-425d-a543-561fb751c088)

With this I think we would have left this in a good enough state to focus on other stuff.

gtk-grafana commented:
> Changing the control to be slightly more verbose.
> Only show either version of "Show errors" when there are actually panels with errors

Added in https://github.com/grafana/logs-drilldown/pull/1346/commits/4561b0b69dccc3424ab1c033bb4418d1b4b98249

gtk-grafana commented:
> Disable or gray out the "select" button on failed panels?
> The field breakdown looks currently broken. One thing that we're missing there is a button to return to the fields/label breakdown.

Added in https://github.com/grafana/logs-drilldown/pull/1346/commits/f244fc01b1df8e2db72ae8b7798e1ced8bd59693

@matyax if you want to address the max series error message in the panel in another branch I think this one is ready for another look. Sorry about f244fc01b1df8e2db72ae8b7798e1ced8bd59693, it was a big-un.

Oh, I also removed the "single" layout because it's already buggy/broken in main, if we want to support the single view we should support it, but it being in a half supported state isn't good for anyone, plus with the new summary panel that is actually filterable on interaction I don't really see the purpose of the single view in values any more.

matyax commented on src/Components/ServiceScene/Breakdowns/LayoutSwitcher.tsx:
```diff
@@ -16,10 +16,9 @@ export interface LayoutSwitcherState extends SceneObjectState {
   options: Array<SelectableValue<LayoutType>>;
 }
 
-export type LayoutType = 'grid' | 'rows' | 'single';
```
👌
